.github/docker/Dockerfile.centreon-collect-mysql-alma9

```diff
@@ -48,6 +48,8 @@ dnf install -y cmake \
     nfpm \
     sudo
 
+dnf upgrade python3
```

Comment:
python3 should be already on latest version because it is installed just before this step

Reply:
Je viens de l'enlever et voilà:
#7 50.78 ERROR: Exception:
#7 50.78 Traceback (most recent call last):
#7 50.78   File "/usr/lib/python3.9/site-packages/pip/_internal/utils/unpacking.py", line 198, in untar_file
#7 50.78     tarfile.data_filter(member.replace(name=fn), location)
#7 50.78 AttributeError: module 'tarfile' has no attribute 'data_filter'
#7 50.78 
#7 50.78 During handling of the above exception, another exception occurred:
#7 50.78

Reply:
then use ` dnf --best install` in the previous command instead of `dnf install`
this will install the latest version of python3